metis-validation/metis-validation-rest/src/main/resources/schema/RDF.xsd

```diff
@@ -183,4 +183,73 @@
 			</extension>
 		</simpleContent>
 	</complexType>
-</schema>
\ No newline at end of file
+
+	<complexType name="RightsStatements">
+		<attribute name="resource" use="required" form="qualified">
```

Comment:
Wouldn't this attribute conflict with the one declared at L146, during validation??

Reply:
not, it will be cumulative... the rights statements are still uris so we are good.

Reply:
Since you've verified this then it's fine